lib/Access.php

```diff
@@ -1991,4 +1990,46 @@ private function initPagedSearch($filter, $bases, $attr, $limit, $offset) {
 
 		return $pagedSearchOK;
 	}
+
+	/**
+	 * Retries paged LDAP search without handling from 0 to target offset with limit
+	 *
+	 * @param string $filter the LDAP filter for the search
+	 * @param array $base an array containing the LDAP subtree(s) that shall be searched
+	 * @param null|string|string[] $attr
+	 * @param int $limit
+	 * @param int $targetOffset
+	 * @return boolean if retry of paged search has been successful
+	 * @throws \OC\ServerNotAvailableException
+	 */
+	private function reExecutePagedSearch($filter, $base, $attr, $limit, $targetOffset) {
+		if ($targetOffset % $limit != 0) {
```

Comment:
`!==` or include a comment if it's intended

Reply:
Actually I moved this function to `search()`, it was too confusing